### PR TITLE
Implementation docs

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi @jkortega9008, @ramniel, @ms-darianbenito, and @MarianoHerrera,

I am adding the basic implementation details related to the interaction with the Doppler database, along with a little database script to generate the Doppler DB subset that will be used by this project.

For free, I have also updated the file `demo.http` to test this API easily in any environment.

Could you kindly review it?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Doppler Contact Policies API
+
+Mini-API to allow users to configure Doppler's contact policies.
+
+See [implementation details](./docs/implementation-details.md).
+
+You can test is using [VS Code REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) with [demo.http](./demo.http).
```
Comment (ms-darianbenito) [inline README.md:7]: What do you think about including the following information?

```suggestion
You can test is using [VS Code REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) with [demo.http](./demo.http).

# CD
- Production environment
URL: https://apis.fromdoppler.com/contact-policies 
Updated when generating a new tag.

- Integration environment
URL: https://apisint.fromdoppler.net/contact-policies 
Updated when add changes into the INT branch

- QA environment
URL: https://apisqa.fromdoppler.net/contact-policies 
Updated when add changes to the main branch
```

Comment (andresmoschini) [reply README.md:7]: Mmm, ok.

```suggestion
You can test is using [VS Code REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) with [demo.http](./demo.http).

## Environments 
          
### Production

URL: <https://apis.fromdoppler.com/contact-policies/>

The production environment is updated when a new `v1.*` tag is pushed.

### QA

URL: <https://apisqa.fromdoppler.net/contact-policies/>

The QA environment is updated when `main` branch is updated.

### INT

URL: <https://apisint.fromdoppler.net/contact-policies/>

The INT environment is updated when `INT` branch is updated.          
```

Comment (andresmoschini) [reply README.md:7]: Thanks @ms-darianbenito, I updated it based on your comment, with some format and style changes. Could you review it again?

File: demo.http [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
 
-###
+### See current version
 GET {{base}}/version.txt
+
+@accountName = test
+@token = test
+
+### Get account's settings
+GET {{base}}/accounts/{accountName}/settings
```
Comment (ms-darianbenito) [inline demo.http:16]: 🤔 
```suggestion
GET {{base}}/accounts/{{accountName}}/settings
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+URL: <https://apisqa.fromdoppler.net/contact-policies/>
+
+The QA environment is updated when `main` branch is updated.
+
+### INT
+
+URL: <https://apisint.fromdoppler.net/contact-policies/>
+
+The INT environment is updated when `INT` branch is updated.
```
Comment (ms-darianbenito) [inline README.md:27]: I known, the following information is in `demo.http` file but, what do you think?

```suggestion

## Environments

### Production

URL: <https://apis.fromdoppler.com/contact-policies/>

The production environment is updated when a new `v1.*` tag is pushed. 

### QA

URL: <https://apisqa.fromdoppler.net/contact-policies/>

The QA environment is updated when `main` branch is updated.

### INT

URL: <https://apisint.fromdoppler.net/contact-policies/>

The INT environment is updated when `INT` branch is updated.

You can check the environment´s current version using [ENVIRONMENT_URL]/version.txt
```

File: demo.http (hunk 2 above)
Comment (andresmoschini) [reply demo.http:16]: Good catch!

File: README.md (hunk 1 above)
Comment (ms-darianbenito) [reply README.md:7]: Great!

**Comment by ms-darianbenito** [comment]
👍

File: README.md (hunk 3 above)
Comment (andresmoschini) [reply README.md:27]: Well, it is out of the original scope of this PR, for the moment I prefer keeping it as it is.

File: docs/implementation-details.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+### Tabla `UserShippingLimit`
+
+Contiene la configuración básica de políticas de contacto. Si no existe o `UserShippingLimit.Active = 0` se considera que está desactivado.
+
+**Campo `UserShippingLimit.Amount`:** Define la cantidad de emails a enviar en el período de tiempo.
+
+**Campo `UserShippingLimit.Interval`:** Define el período de tempo medido en días.
+
+Por el momento, si se intenta a la edición de políticas de contacto y la fila `UserShippingLimit` relacionada al usuario no existe, se podría manejar como un error.
```
Comment (ms-darianbenito) [inline docs/implementation-details.md:19]:
```suggestion
Por el momento, si se intentan editar las políticas de contacto y la fila `UserShippingLimit` relacionada al usuario no existe, se podría manejar como un error.
```

File: docs/implementation-details.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+### Política de contacto desactivada
+
+[Ver mockup](https://www.figma.com/file/YjtGdoOSCoo4UTTtGxZowV/Pol%C3%ADtica-de-Contacto?node-id=205%3A4080)
+
+Cuando:
+
+- tiene permisos (Ver [Doppler Users API](https://github.com/FromDoppler/doppler-users-api)) y:
+  - cuando no existe la fila `UserShippingLimit`
+  - o cuando `UserShippingLimit.Active = 0`
```
Comment (ms-darianbenito) [inline docs/implementation-details.md:57]: Ese espacio hace que se vea así:
image


```suggestion
- tiene permisos (Ver [Doppler Users API](https://github.com/FromDoppler/doppler-users-api)) y:
- cuando no existe la fila `UserShippingLimit`
- o cuando `UserShippingLimit.Active = 0`
```

Comment (ms-darianbenito) [reply docs/implementation-details.md:57]: Ahí vi los dos puntos. Disculpas

File: docs/implementation-details.md (hunk 4 above)
Comment (andresmoschini) [reply docs/implementation-details.md:19]:
```suggestion
Por el momento, si se intenta editar las políticas de contacto y la fila `UserShippingLimit` relacionada al usuario no existe, se podría manejar como un error.
```

**Pull request merged**